README.md

```diff
@@ -137,18 +137,48 @@ Events:
 - Status
 - Workflow job
 
-# Development
+# Local development
 
-The bot requires some environment variables listed in
-[config.rs](./src/config.rs). They can, optionally, be set through an `.env`
-file which should be placed at the root of this repository. We provide an
-[example .env file](./.env.example) which can be used as a starting point.
+This project is a standard Rust project with some notable requirements.
+
+## Requiremnets
+
+Before you can generate a debug or release binary, we have some library
+requirements to install.
+
+```sh
+$ sudo apt install \
+    libssl-dev \
+    libclang-dev
+```
+
+## Environment variables
+
+This bot relies on the crate `dotenv`, which requires a `.env` file to
+exist in the root of the project that the binary runs in. These environment
+variables are listed in [config.rs](./src/config.rs).
+
+Copy over the example environment file to suite your needs like so
```

Comment:
It would be better to have the link to the `.env.example` file as in the original text